wordpress-mobile/WordPress-iOS
# Add "This Week" localized string to StatsWidgets target

Fixes #NA

To test:
- Build/run and install "This Week" widget
- make sure "This Week" title shows up correctly

**Note:**
At this time, translations are not present so we can only test the correctness in the base language (en)

cc @jkmassel 

PR submission checklist:
